dags.yaml

```diff
@@ -838,3 +838,23 @@ bqetl_telemetry_crashes_daily:
   tags:
     - impact/tier_2
     - repo/bigquery-etl
+
+bqetl_internal_tooling_wchan:
+  default_args:
+    depends_on_past: false
+    email:
+      - telemetry-alerts@mozilla.com
+      - wichan@mozilla.com
+    email_on_failure: true
+    email_on_retry: true
```

Comment:
I'd suggest to only alert on failure, especially since tagged as `tier_3`

Reply:
I have removed email_on_retry